tracer/test/Datadog.Trace.TestHelpers/ModuleInitializerAttribute.cs

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-#if !NET5_0
+#if !NET5_OR_GREATER
```

Comment:
NET5_0_OR_GREATER

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2241/commits/d9413e5b83ed665c1bb27152adc038981b556dbe